rswag-specs/lib/rswag/specs/example_group_helpers.rb

```diff
@@ -52,10 +54,21 @@ def parameter(attributes)
       end
 
       def response(code, description, metadata = {}, &block)
-        metadata[:response] = { code: code, description: description }
+        metadata[:response] = { code: code, description: description, examples: {} }
         context(description, metadata, &block)
       end
 
+      def example_context(description, metadata = {}, &block)
+        context(description, metadata.merge(skip_formatter: true), &block)
+      end
+
+      def example_value(value)
+        metadata[:response][:examples][metadata[:description].parameterize separator: '_'] = {
```

Comment:
In `ActiveSupport` < 5 this method had different signature: `parameterize(sep = '-')`
And since we [support](https://github.com/BookOfGreg/rswag/blob/10bb73214882c019d91e0e48d01e6aacaa03ceb1/rswag-specs/rswag-specs.gemspec#L16) versions >= 3.1, I suggest to drop the argument and stick with the default separator.